changelogs/fragments/2036-missing_error_msg_in_vmware_guest.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+  - Fix a missing error message for the module `vmware_guest` while setting a template parameter with inconsistency guest_os ID.
```

```suggestion
bugfixes:
  - vmware_guest - Fix a missing error message while setting a template parameter with inconsistency guest_os ID
    (https://github.com/ansible-collections/community.vmware/pull/2036).
```

It helps people reading the changelog if you say first what module has been changed. At least, if it's only one module. If the change affects several modules, a complete list might be too confusing.

Adding a link to the issue (or, if there's no issue like in this case, to the PR) might help people to find more information about the change. I think it's always been a link to the issue if one exists and a link to the PR otherwise. But I'm not 100% sure if this is still best practice today. Still looks sensible to me, although I'm biased... as I said: That's how I know it.

---

Thank you for sharing your knowledge with me it makes sense.
Fixed it as you suggested